.travis.yml

```diff
@@ -6,7 +6,10 @@ cache:
 notifications:
   email: false
 node_js:
+  - 4
+  - 5
```

Comment:
Testing in 5 and 7 implies we're able to maintain support with those versions. I would suggest LTS (4 & 6 is enough).